student-software-developers/administration-handbook/timeline/02-february.md

```diff
@@ -0,0 +1,27 @@
+### Early February: Perform search
+
+#### (Optional) 1st week of February: Host a table at the Student Employment Office’s Spring Job Fair
+
+We did this in 2023. 11 people signed our interest form, but none of them applied for either of our positions, so it may not make sense to do this again.
```

Comment:
Maybe move all this text to the recruitment page, so that reading that page gives a full sense of the things we have done / could do for recruiting? We should leave the heading, though, since if we wanted to do it we'd need that timeline reminder.